core/src/scans.rs

```diff
@@ -123,6 +123,10 @@ pub fn register_pool_box_scan(oracle_pool_nft: &TokenId) -> Result<Scan> {
             "predicate": "containsAsset",
             "assetId": oracle_pool_nft.clone(),
         },
+        {
+            "predicate": "containsAsset",
```

Comment:
With pool token being NFT (only one exists) I don't see how there could be more than one box with the same pool NFT anyway. Are you sure we need this reward token id filter?

Reply:
Yes of course, thanks for that, we don't need it.